### DISCUSSION TITLE
[CHANGE] How will procurement officers be trained

### DISCUSSION THREAD

**Discussion by mgifford** [gc-da11yn/gc-da11yn.github.io#240] [Standard on Information and Communication Technology (ICT) Accessibility / Norme]
Looking at https://a11y.canada.ca/en/standards/standard/#2.3.1.1

2.2 This standard applies to all information and communication technology (ICT) that is designed, procured, and used by the Government of Canada (GC).

Procurement officers in the GC have 20 years where WCAG requirements were included in the boilerplates for their contracts. For 20 years it has been clear that this is a "nice to have". Switching from, "The contract deliverables must meet WCAG 2.0 AA" to "The contract deliverables must meet the latest EN 301 549 standard" will accomplish next to nothing on its own.

Yes, it is needed. Some global best practices can be found here https://github.com/mgifford/a11y-contracting

**Comment by jrowsell** [comment]
This section is not provide enough context or detail. Please consider the readability and understandability of this policy and that many of the people who will be asked to follow it will not be policymakers. 

It's okay to rely on a standard set elsewhere, but without supporting guidance this cannot and will not be applied across GC. We still have to train procurement teams on HOW to evaluate for accessibility. We have seen time and time again inaccessible solutions from vendors because of the language used in procurement. Examples of this language and rubric for baseline testing is important. 

In addition, procurements where an RFP is accepted with exceptions or partial compliance because it's on a vendor's roadmap should have accountability gates. What we see in practice right now even with accessibility standards is enterprise solutions with promises that do not get delivered and thus hacks and workarounds stunt innovation and reduce agency and autonomy. By the time a contract expires that status is still partial compliance or non-compliance and we go through the process all over again. Vendor's feel little to no pressure to deliver on accessibility. 

**I encourage you to think on the following questions: **

1, What is the accountability framework?
2. What exceptions might exist? How will risk be reduce and mitigated?
3. How will procurements be evaluated in the RFP and RFI process?
4. Will selected vendors been held accountable to their roadmap exceptions that allow a VPAT that is partial compliance to be remedied?
5. Are their groups connecting across the GC with more empowerment? AWG **does not** get enough **_support_**.
6. How will you publicly report on overall compliance?
7. How will this tie with departmental requirements to meet the ACA?

Linkages to guidance, standards and directives are important here. Ideally there would be a learning offering created and delivered that is mandatory for all procurement officers, procurement evaluators, and feedback loop opportunities for all users internal and external to the GC.
